.github/workflows/bandit.yml

```
@@ -24,4 +24,4 @@ jobs:
       - name: Analysing the code with bandit
         run: |
           echo $PWD
-          bandit -a file $(git ls-files '*.py') 
+          bandit -a file $(git ls-files 'src/*.py') 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)